webapi/src/main/scala/org/knora/webapi/store/triplestore/embedded/JenaTDBActor.scala

```diff
@@ -132,9 +132,10 @@ class JenaTDBActor extends Actor with ActorLogging {
       * Submits a SPARQL query to the embedded Jena TDB store and returns the response as a [[SparqlSelectResponse]].
       *
       * @param queryString the SPARQL request to be submitted.
+      * @param useInference not supported.
```

Comment:
GraphDB accepts an `infer` HTTP parameter to enable or disable inference. The `useInference` parameter is used for that. However, it is not sufficient, because setting `infer=false` does not turn off the [owl:sameAs optimisation](http://graphdb.ontotext.com/documentation/standard/sameas-optimisation.html). Therefore, to completely disable inference for a query in GraphDB, we must also use `FROM <http://www.ontotext.com/explicit>` in the SPARQL.